tests/test_models.py

```diff
@@ -19,7 +19,7 @@ def tearDown(self):
         pass
 
     def __create_cat(cls, name, parent=None, priority=0):
-        return Category.objects.create(
+        return cls.CATEGORY_MODEL.objects.create(
```

Comment:
If this method needs to access to `self._category_model` instance property, it can't be a `@staticmethod` anymore.